helm/identity-account-mapper/account-mapper/values.yaml

```diff
@@ -9,8 +9,7 @@ configmap:
 
 enabled: true
 replicas: 1
-image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/ph-ee-identity-account-mapper"
-imagetag: latest
+image: docker.io/openmf/ph-ee-identity-account-mapper:v1.2.0
```

Comment:
This should point to latest image by default 
if the tagged version is used in ph-ee-engine, then g2p samdox chart should point to laters 
The idea is to enable the chart for CI run